lib/database_cleaner/configuration.rb

```diff
@@ -13,6 +13,19 @@ def [](orm, opts = {})
       fetch([orm, opts]) { add_cleaner(orm, opts) }
     end 
 
+    def strategy
```

Comment:
@rlue I think it would be great if you can document this method. It might be unexpected for users that the `strategy` method can return nil, a strategy object, or an array.